### ISSUE TITLE
Needed: More specific guidance for buying groceries.

### ISSUE THREAD

**Issue by jmcmurry** [flattenthecurve/guide#151]
See [here](https://twitter.com/vsbuffalo/status/1240300326516715525).

"I've drastically reduced grocery story visits by stocking up (ages ago, when people thought I was crazy) and making dishes go further with rice, but what's recommended when it comes to grocery shopping? Continue to go rarely? Go during off hours? Avoid going entirely?"

Some good suggestions in comment thread.

**Comment by nicolevasilevsky** [comment]
Some grocery stores are now offering 'elderly/vulnerable only hours' too:
https://www.businessinsider.com/coronavirus-stores-special-hours-elderly-vulnerable-list-2020-3

**Comment by jmcmurry** [comment]
Yes; I'm also developing some ideas [here](https://docs.google.com/document/d/1SygSVuAvQJAHXNxFU9IH5b3I7fBoEsfuHE1M99_fnFY/view).
